web/react/components/user_settings/user_settings_general.jsx

```diff
@@ -570,6 +585,76 @@ export default class UserSettingsGeneralTab extends React.Component {
                 />
             );
         }
+
+        var miscellaneousSection;
+        var describeCtrlSend = this.state.props.ctrlSend === 'true' ? 'On' : 'Off';
+        if (this.props.activeSection === 'ctrlSend') {
+            var ctrlSendActive = [false, false];
+            if (this.state.props.ctrlSend === 'true') {
+                ctrlSendActive[0] = true;
+            } else {
+                ctrlSendActive[1] = true;
+            }
+
+            let ctrlSendInputs = [];
+            ctrlSendInputs.push(
+                <div
+                    key='ctrlSendSetting'
+                    className='form-group'
+                >
+                    <div className=''>
+                        <div className='radio'>
+                            <label>
+                                <input
+                                    type='radio'
+                                    onChange={() => this.updateCtrlSend('true')}
+                                    checked={ctrlSendActive[0]}
+                                />
+                                {'On'}
+                            </label>
+                            <br/>
+                        </div>
+                        <div className='radio'>
+                            <label>
+                                <input
+                                    type='radio'
+                                    onChange={() => this.updateCtrlSend('false')}
+                                    checked={ctrlSendActive[1]}
+                                />
+                                {'Off'}
+                            </label>
+                            <br/>
+                        </div>
+                        <div><br/>{'If enabled \'Enter\' inserts a new line and ctrl + enter submits the message.'}</div>
```

Comment:
Proposed edit: `'If enabled \'Enter\' inserts a new line and \'Ctrl + Enter\' submits the message.'`